### PR TITLE
add faq-environments

### PR DESCRIPTION
**Pull request by MelissaGraham**

### CODE REVIEW

**Review by leannep — APPROVED** [review]
Looks good. I tested this on data.lsst.cloud

File: data-access-analysis-tools/nb-intro.rst [hunk 1]
```diff
@@ -176,6 +176,30 @@ or this Community Forum thread where DP0 delegates can share `resources for pyth
...
+If you need to extend the ``rubin-env`` environment by installing other Python packages to enable your work, you can use the ``pip install`` command.
+In the RSP, ``pip`` actually invokes ``conda`` to do its work, ensuring that dependencies that are already present in ``rubin-env`` are used (if compatible).
+Packages installed with ``pip`` will be placed in a subdirectory of your home directory.
+These packages are only guaranteed to work when the conda environment in which you installed them is activated.
+
+If you need to install other conda packages but don't need to use them at the same time as the ``rubin-env`` and LSST Science Pipelines packages, you can install them into a new conda environment.
+Start by doing ``source /opt/lsst/software/stack/loadLSST.bash`` to initialize conda.
+Use the ``conda create -n myenv`` command to create the new environment.
+Use the ``conda activate myenv`` command to activate this environment.
+Use the ``mamba install {package} ...`` command to install one or more packages into the environment.
```
Comment (leannep) [inline data-access-analysis-tools/nb-intro.rst:194]: One point maybe to add is that if the package to be installed is not  available from current channels the channel must be specified, e.g 
> ``mamba install -c pyviz lumen``

**Review by leannep — CHANGES_REQUESTED** [review]

File: data-access-analysis-tools/nb-intro.rst (hunk 1 above)
Comment (MelissaGraham) [reply data-access-analysis-tools/nb-intro.rst:194]: Thanks! Added:

If the package to be installed is not available from the current channels, then the channel will have to be specified, e.g., ``mamba install -c pyviz lumen``.

File: data-access-analysis-tools/nb-intro.rst [hunk 2]
```diff
@@ -176,6 +176,31 @@ or this Community Forum thread where DP0 delegates can share `resources for pyth
...
+(``mamba`` is a faster version of conda for installing packages.)
+If the package to be installed is not available from the current channels, then the channel will have to be specified, e.g., ``mamba install -c pyviz lumen``.
```
Comment (leannep) [inline data-access-analysis-tools/nb-intro.rst:196]: replace "lumen" with "{package} and "pyviz" with {channel}. - this was the specific example I used to test but I wouldn't want to look like we are suggesting a specific package

**Review by leannep — CHANGES_REQUESTED** [review]

File: data-access-analysis-tools/nb-intro.rst (hunk 2 above)
Comment (MelissaGraham) [reply data-access-analysis-tools/nb-intro.rst]: makes sense. updated.

**Review by leannep — APPROVED** [review]

**Pull request merged**